chill-scala/src/main/scala/com/twitter/chill/ScalaKryoInstantiator.scala

```diff
@@ -121,15 +98,32 @@ class ScalaCollectionsRegistrar extends IKryoRegistrar {
      */
     newK
       // wrapper array is abstract
+      .registerClasses(Seq(
```

Comment:
can you add all new items at the end of the list? Otherwise, I think we could change the tokens and some people have (ill-advisedly) persisted kryo data into long term storage.

Reply:
yes... I will follow your next proposal and put all new items into a separate registrar.